src/server/endpoints/covidcast_utils/db_signals.csv

```diff
@@ -1,29 +1,32 @@
+dsew-cpr,confirmed_admissions_covid_1d_7dav,TRUE,confirmed_admissions_covid_1d_prop_7dav,FALSE,Confirmed COVID Admissions per day (per 100k people),FALSE,,,COVID-19 Community Profile Report,covid,Vaccines,USA,"county,msa,state,hhs,nation",nation,2020-12-16,"Start dates vary by geo: county 2021-01-07, hhs 2020-12-16, msa 2021-01-07, nation 2020-12-16, state 2020-12-16",2023-02-21,NA,day,Date,daily,3-9 days,Daily. This data source is susceptible to large corrections that can create strange data effects such as negative counts and sudden changes of 1M+ counts from one day to the next.,All,None,hospitalized,None,Data is available for all counties and some territorial county equivalents. Data is available for all states and some territories.,Value,per100k,late,bad,TRUE,FALSE,FALSE,FALSE,FALSE,public,public,Public Domain US Government (https://www.usa.gov/government-works),See license,NA,https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/dsew-cpr.html
+dsew-cpr,covid_naat_pct_positive_7dav,FALSE,covid_naat_pct_positive_7dav,FALSE,Rate of COVID NAAT Test Positivity,FALSE,Rate of COVID NAAT test positivity,"âNAAT positivity rate is the number of positive [SARS-CoV-2 Nucleic Acid Amplification Test (NAAT)] tests divided by the number of [NAAT] tests performed and resulted. See the CDCâs coronavirus resources page for more information on this method. In rare cases, negative test numbers may be reported on a given day to correct previous numbers, and this sometimes results in a calculated test positivity of greater than 100%. In these cases, test positivity is capped at 100%.â - from the CPR data dictionary.
+
+This is reported as a rate, between the values of 0 and 1, not a percent.",COVID-19 Community Profile Report,covid,Testing,USA,"county,msa,state,hhs,nation",nation,2020-12-07,NA,2023-02-20,"End dates vary by geo: county 2023-02-13, hhs 2023-02-20, msa 2023-02-20, nation 2023-02-20, state 2023-02-13",day,Date,daily,3-9 days,Daily. This data source is susceptible to large corrections that can create strange data effects.,All,None,infected,"""In rare cases, negative test numbers may be reported on a given day to correct previous numbers, and this sometimes results in a calculated test positivity of greater than 100%. In these cases, test positivity is capped at 100%.â - from the CPR data dictionary.",Data is available for all counties and some territorial county equivalents. Data is available for all states and some territories.,Percentage,fraction,cases_testing,bad,TRUE,FALSE,FALSE,FALSE,FALSE,public,public,Public Domain US Government (https://www.usa.gov/government-works),See license,NA,https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/dsew-cpr.html
```

Comment:
There are a few extended ascii characters added here that give me pause.  They appear to be for/from fancy quotation marks.  I changed them in the source spreadsheet, so we should do another sync of this csv file before we release anything to production.